Makefile

```diff
@@ -50,7 +50,7 @@ ensure-venv:
 	@./scripts/ensure-venv.sh
 
 ensure-pinned-pip:
-	$(PIP) install --upgrade pip
+	$(PIP) install --no-cache-dir "pip>=20.0.2"
```

Comment:
You may still wanna keep the `--upgrade` flag to force the latest available version I think. Not sure how it interacts with version ranges.

Reply:
```
joshua.li@tim-apple joshua.li $ pip --version
pip 19.2.3 from /Users/joshua.li/.local/share/pyenv/versions/3.7.4/lib/python3.7/site-packages/pip (python 3.7)
joshua.li@tim-apple joshua.li $ pip install --no-cache-dir 'pip>=20.0.2'
Collecting pip>=20.0.2
  Downloading https://files.pythonhosted.org/packages/54/0c/d01aa759fdc501a58f431eb594a17495f15b88da142ce14b5845662c13f3/pip-20.0.2-py2.py3-none-any.whl (1.4MB)
     |████████████████████████████████| 1.4MB 1.7MB/s
Installing collected packages: pip
  Found existing installation: pip 19.2.3
    Uninstalling pip-19.2.3:
      Successfully uninstalled pip-19.2.3
Successfully installed pip-20.0.2
```